service/service.go

```diff
@@ -51,15 +51,15 @@ func Run(ctx context.Context, cfg *config.Config, serviceList *ExternalServiceLi
 		S3UploadBucket: cfg.UploadBucketName,
 		S3Interface:    s3Client,
 		Producer:       producer,
-	}, cfg.KafkaConsumerWorkers)
+	}, cfg.KafkaConsumerWorkers)*/
 
 	//heathcheck - start
 	hc, err := serviceList.GetHealthCheck(cfg, buildTime, gitCommit, version)
 	if err != nil {
 		log.Fatal(ctx, "could not instantiate healthcheck", err)
 		return nil, err
 	}
-	if err := registerCheckers(ctx, cfg, hc, producer, consumer, s3Client); err != nil {
+	if err := registerCheckers(ctx, cfg, hc, nil, nil, nil); err != nil {
```

Comment:
i think we might need to comment out in registerCheckers too? maybe not though - does it report healthy when you start?